docs/contributors/release.md

```diff
@@ -1,162 +1,142 @@
+### Release Candidates
 
-# Release Candidates
+On the date of the current's milestone, we publish a release candidate and make it available for plugin authors and for users to test. If any bugs/regressions are found with a release candidate, a new release candidate can be published.
 
-Since [`3.8`](https://github.com/WordPress/gutenberg/releases/tag/v3.8.0), we [publish a release candidate](https://github.com/WordPress/gutenberg/releases/tag/v3.8.0-rc.1) at least one week before we publish a new version. This release candidate is available for plugin authors and for users to test. If any bugs/regressions are found with a release candidate, they should be cherry-picked into a new release candidate and it should be published.
+
+The date in the milestone is the date of **tagging the release candidate**. On this date, all remaining PRs on the milestone are punted automatically to the next release.
 
 Release candidates should be versioned incrementally, starting with `-rc.1`, then `-rc.2`, and so on.
 
-## Creating a Release Candidate
+#### Creating the first Release Candidate
 
-Creating a release candidate involves:
+Releasing the first release candidate for this milestone (`x.x`) involves:
 
 1. writing a release blog post and changelog
-2. bumping the version
-3. tagging the release
-4. building the plugin
-5. publishing the new release to GitHub
-6. committing to the [plugin repository]
+2. creating the release branch
+3. bumping the version and tagging the release
+5. building the plugin
+6. publishing the release to GitHub
+7. publishing the call for testing
 
-### Writing the Release Post and Changelog
+##### Writing the Release Post and Changelog
 
 1. Open the [list of closed pull requests](https://github.com/WordPress/gutenberg/pulls?utf8=✓&q=is%3Apr+is%3Aclosed+sort%3Acreated-desc+) and filter by the current milestone.
 2. Read through each PR  to determine if it needs to be included in the blog post and/or changelog.
 3. Choose a few features to highlight in the release post; record an animation of them in use.
 4. Save the draft post on [make.wordpress.org/core](https://make.wordpress.org/core/); this post should be published after the actual release.
 
-### Bumping the Version
+##### Creating the Release Branch
 
-1. Create [a pull request like this](https://github.com/WordPress/gutenberg/pull/9663), bumping the version number in `gutenberg.php`, `package.json`, and `package-lock.json`.
-2. Check that there's no work-in-progress that's just about to land. [Inform committers in `#core-editor` on Slack](https://wordpress.slack.com/messages/C02QB2JS7) to hold off on merging any changes until after the release candidate is tagged.
-3. Merge the version bump pull request.
+For each milestone (let's assume it's `x.x` here), a release branch is used to release all RCs and minors. For the first RC of the milestone, a release branch is created from master.
```

Comment:
```suggestion
For each milestone (let's assume it's `x.x` here), a release branch is used to release all RC, minor, and bugfix releases. For the first RC of the milestone, a release branch is created from master.
```

Reply:
Not sure about this one as we don't have a concept of minor and bug fixes. We only have major and minor (for Gutenberg and Core).

Reply:
ah, fair enough. Maybe just "minor releases" instead of "minors". Just "minors" seemed unclear to me.